src/modules/75-cf/strings/strings.en.yaml

```
@@ -505,6 +505,16 @@ pipeline:
     serve: Serve
     serveVariation: Serve variation
     flagChanges: Flag Changes
+    flagChangesV2Runtime: >
+      <p>At runtime, you may choose the following flag configuration changes:</p>
+      <ul>
+        <li>Set flag switch</li>
+        <li>Set default ON rule</li>
+        <li>Set default OFF rule</li>
+        <li>Serve variation to targets</li>
+        <li>Serve variation to target groups</li>
+        <li>Serve percentage rollout to target group</li>
```

```suggestion
        <li>Serve percentage rollout to target groups</li>
```
should this be plural?

---

Nope, you can only set for a single target group. Funnily enough, I had it as plural on accident and only spotted the error just before I committed.